ansible-ipi-install/roles/installer/tasks/57_check_vip_status.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Test the connectivity for {{ apivip }} 
+  command: ping -c3 -i 3 {{ apivip }} 
+  register: apivip_results
+  tags: vip_status
+
+- name: Test the connectivity for {{ ingressvip }}
+  command: ping -c3 -i 3 {{ ingressvip }}
+  register: ingressvip_results
+  tags: vip_status
+
+- name: Print the ping results for {{ apivip }}
+  debug:
+    var: apivip_results.stdout
+    verbosity: 2
+  tags: vip_status
+
+- name: Print the ping results for {{ ingressvip }}
+  debug:
+    var: ingressvip_results.stdout
+    verbosity: 2
+  tags: vip_status
+
+- name: If VIP's are being used fail with an error message
```

Comment:
Let's change name to "Fail if API VIP or Ingress VIP are active"

Reply:
Side note, can you squash all commits down to 1 commit?

Reply:
I just added all the suggested changes and also squashed it. Thanks.

Reply:
> I just added all the suggested changes and also squashed it. Thanks.

Thanks for the changes. However, the commits still show 6, we want to squash them down to 1.